### PR TITLE
Throw LogViewerException instead of generic exception in restoreDefaults method

### PR DESCRIPTION
**Pull request by ChanakaCooray**

### CODE REVIEW

File: components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java [hunk 1]
```diff
@@ -442,7 +442,7 @@ public static void restoreDefaults() throws Exception {
         } catch (FileNotFoundException e) {
             String msg = "Cannot restore default logging configuration."
                     + " log4j.properties file not found in the classpath";
-            throw new Exception(msg, e);
+            throw new LogViewerException(msg, e);
```
Comment (nishadi) [inline components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java]: Is there any specific reason for creating a different variable for the exception message?

File: components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java [hunk 2]
```diff
@@ -442,7 +442,7 @@ public static void restoreDefaults() throws Exception {
         } catch (FileNotFoundException e) {
             String msg = "Cannot restore default logging configuration."
                     + " log4j.properties file not found in the classpath";
```
Comment (nishadi) [inline components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java]: Shall we fix the grammar issue as to ;
" log4j.properties file **is** not found in the **classpath.**"

File: components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java (hunk 1 above)
Comment (ChanakaCooray) [reply components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java]: @nishadi this is the pattern we have followed in most of the classes.

File: components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java (hunk 2 above)
Comment (ChanakaCooray) [reply components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java]: Fixed, thanks for pointing out.

**Pull request merged**